gef-ui/server/controllers/submit-to-ukef/index.test.js

```diff
@@ -94,6 +94,7 @@ describe('controllers/submit-to-ukef', () => {
   describe('Ascertain correct deal submission type', () => {
     it('Should stay as an AIN', async () => {
       const submissionType = CONSTANTS.DEAL_SUBMISSION_TYPE.AIN;
+      mockRequest.body.confirmSubmitUkef = 'true';
```

Comment:
is this because the html component is returning the value as a string?

Reply:
yes, it won't return it as boolean